plugins/Marketplace/vue/src/PluginList/HelpLink.vue

```diff
@@ -0,0 +1,31 @@
+<!--
+  Matomo - free/libre analytics platform
+  @link https://matomo.org
+  @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+-->
+
+<template>
+  <a tabindex="7"
+     class="plugin-details"
+     href="#"
+     v-plugin-name="{ pluginName }"
+     :title="translate('General_MoreDetails')"
+  >{{ translate('General_Help') }}</a>
```

Comment:
These two strings probably should be the same? Either More details or Help.

Reply:
Yes and no :thinking: 

It was always displayed in this combination. And the link is not really a "Help", it only opens the "More Details" page as it is getting caught at the card level (and rerouted to the title link).